pkg/container/types/time.go

```diff
@@ -168,6 +173,66 @@ func ParseTime(s string, precision int32) (Time, error) {
 	return FromTimeClock(isNegative, int32(hour), uint8(minute), uint8(sec+uint64(carry)), msec), nil
 }
 
+// Numeric 112233/112233.4444 should be treate like string and then
+// cast into time type
+// The integrity numeric is int64 while numeric with decimal is Decimal128
+// The number befre the decimal point is the hh:mm:ss part while
+// decimal part is the msec.
+// eg.
+//
+//	112233 -> "0112233" -> "11:22:33"
+//	123 -> "0000123" -> "00:01:23"
+//	112233.444 -> "112233.444" -> "11:22:33.444"
+
+func ParseInt64ToTime(input int64, precision int32) (Time, error) {
+	// the input must should in range [-838:59:59, 838:59:59]
+	if input >= 8390000 || input <= -8390000 {
```

Comment:
What is 838?   Where does it come from?

Reply:
It is from MySQL standard which seems that they use for backward compatibility with the older version(the largest second number that 3 bytes can represent)

Reply:
This is rediculous.   Can we just remove this?   Anything will be wrong?

Reply:
We do not arbitrary narrow our types.    There is no reason to do that.